ACHNBrowserUI/ACHNBrowserUI/views/turnips/charts/TurnipsChart.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  TurnipsChart.swift
+//  ACHNBrowserUI
+//
+//  Created by Renaud JENNY on 03/05/2020.
+//  Copyright © 2020 Thomas Ricouard. All rights reserved.
+//
+
+import SwiftUI
+import Backend
+
+// TODO: refactor by renaming this extension to TurnipsChart namespace
+enum TurnipsChart {
+    enum PredictionCurve: String {
+        case minBuyPrice
+        case average
+        case minMax
+
+        var color: Color {
+            // TODO: replace by proper colors
```

Comment:
@Bonney for this one, I think I will need your help.

Reply:
Done